src/discussions/topics/TopicsView.jsx

```diff
@@ -32,7 +32,7 @@ function TopicsView() {
         id={topicGroup.name}
         name={topicGroup.name}
         subtopics={topicGroup.children}
-        key={topicGroup.name}
+        key={topicGroup.id ?? topicGroup.name}
```

Comment:
Yes, topic are in two layers. Parent topics don't have an id only a name, and child topics have both. 
Eventually when we have a new structure for topics this will change, but either way for backwards-compatibility this is needed.